examples/demos/example13-all-features.js

```diff
@@ -196,18 +197,25 @@ export default class extends React.Component {
     return (
       <Wrapper title="All the features grid">
         <Toolbar onAddRow={this.handleAddRow} />
-        <DataGrid
-          ref={node => this.grid = node}
-          enableCellSelect
-          columns={this.columns}
-          rowGetter={this.getRowAt}
-          rowsCount={this.getSize()}
-          onGridRowsUpdated={this.handleGridRowsUpdated}
-          rowHeight={50}
-          minHeight={600}
-          selectedRows={this.state.selectedRows}
-          onSelectedRowsChange={this.onSelectedRowsChange}
-        />
+        <div className="grid-autosizer-wrapper">
+          <AutoSizer>
+            {({ height, width }) => (
+              <DataGrid
+                ref={node => this.grid = node}
+                enableCellSelect
+                columns={this.columns}
+                rowGetter={this.getRowAt}
+                rowsCount={this.getSize()}
+                onGridRowsUpdated={this.handleGridRowsUpdated}
+                rowHeight={30}
+                minWidth={width - 2}
+                minHeight={height - 16}
```

Comment:
Do we need the `- 2` and `- 16`?

Reply:
`width - 2 ` is needed, otherwise you will see the scroll bar is constantly added and removed, height doesn't need to subtract that much, but 16 is a personal preference. I believe for height we at least need to `- 1` for border size.

Reply:
Let me double check, maybe it's due to the height with border.